app/lib/osm-sync.js

```diff
@@ -29,12 +29,14 @@ OsmSync.prototype.replicate = function(target, opts, done) {
   opts.progressFn(0);
 
   function onFinish(err) {
-    console.log("finished replicating", finished);
     if (err) {
       finished--;
       return done(err);
     }
-    if (++finished === 2) done();
+    if (++finished === 2) {
+      console.log("finished replicating", finished);
+      done();
```

Comment:
i think right now we're expecting `progressFn` to get called again. might be good to rearrange it to make it more clear.